src/Avalonia.Controls/Platform/Win32SpecificOptions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Avalonia.Controls;
+using Avalonia.Metadata;
+using Avalonia.Platform;
+using static Avalonia.Controls.Platform.IWin32OptionsTopLevelImpl;
+
+namespace Avalonia.Controls.Platform
+{
+    [Unstable]
```

Comment:
I'm not sure `Unstable` is the right attribute here: both `SetXxxCallback` methods will be marked as obsolete, effectively discouraging or preventing their usage.

Also, the class should probably be made static to prevent inheritance.

Reply:
Class is now static, and Unstable attribute removed